docs/marketplace/dev/architecture-overview/202108.0/marketplace-architecture-overview-domain-model.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Marketplace Domain Model
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace looks like in general. After reading this odc you will now: 
+
+- what are the key Domain Object that makes a Spryker shop to be a Marketplace
+
+## Domain Model
+The following diagram demonstrates the core aggregates of the domain model of the Spyker Marketplace.
+
+Please note that this model is intentionally simplified, learn more about core domain objects in Feature Walkthourghs section.
+
+![Domain Model](https://confluence-connect.gliffy.net/embed/image/02c4183f-2823-4371-ba91-aa5f9585998b.png?utm_medium=live&utm_source=custom)
+
+#### Merchant
+Is a core domain object in Marketplace. Merchant represents an entity of a third-party seller on a shop. Merchants use MerchantPortal to manage their profile, catalog, and sales. 
```

Comment:
not always 3rd party, that could be a fiilials or Drop-shipping partners as well (going back to the Merchant type discussion)